task/build-paketo-builder-oci-ta/0.1/build-paketo-builder-oci-ta.yaml

```diff
@@ -345,13 +356,15 @@ spec:
           mv sbom-temp.json sbom-cyclonedx.json
         fi
 
-        echo "Creating sbom-purl.json"
-        python3 /scripts/create_purl_sbom.py
+        echo "Cat content of the file: parsed_dockerfile.json"
+        cat /shared/parsed_dockerfile.json
+        echo "Check file: sbom-cyclonedx.json"
+        ls -la sbom-cyclonedx.json
```

Comment:
Were these just for debugging or do you want to keep them?

Reply:
Yep. That was there for debug purpose. I will remove them

Reply:
Fixed: f9bff323